router/xgress/retransmitter.go

```diff
@@ -142,10 +147,12 @@ func (retransmitter *Retransmitter) retransmitSender() {
 				if err := retransmitter.forwarder.ForwardPayload(retransmit.x.address, retransmit.payload); err != nil {
 					// if xgress is closed, don't log the error. We still want to try retransmitting in case we're re-sending end of session
 					if !retransmit.x.closed.Get() {
```

Comment:
Would it make sense to put the fault reporting in forward payload for simplicity? Or do we want it here since it will get hit eventually in the retransmit?

Need a way to let the LinkSendBuffer know as well, so it can back off the retransmits

Reply:
I don't think so. It looks like `ForwardPayload` is an interface, so we'd have to implement the fault reporting in everything that implements `ForwardPayload`?

I don't know that I'm super worried about backing off the retransmits for the first push of this. Once the mechanism is working, we can implement those kinds of tweaks, imo...

Reply:
Also, it's the retransmit loop that we need to break. So it seems like the most direct place to put it for now.